### PR TITLE
single pump view implemented and responsive pump list

### PR DESCRIPTION
**Pull request by MatzLeeflang**

### CODE REVIEW

File: Dashboard/resources/js/components/PumpsView/PumpsList.vue [hunk 1]
```diff
@@ -4,15 +4,49 @@ import AlertIcon from 'vue-material-design-icons/Alert.vue';
...
         PumpsListItem,
         AlertIcon,
         CheckboxBlankCircleIcon,
         EyeIcon,
         WrenchIcon
+    },
+     methods: {
+        getPumps(){
+            axios.get(
+                'http://localhost:8000/api/v1/pump',{
```
Comment (Tj33rd) [inline Dashboard/resources/js/components/PumpsView/PumpsList.vue:22]:
```suggestion
               process.env.MIX_API_URL + 'pump', {
```

File: Dashboard/resources/js/components/PumpsView/PumpsList.vue [hunk 2]
```diff
@@ -4,15 +4,49 @@ import AlertIcon from 'vue-material-design-icons/Alert.vue';
...
+                    headers: {
+                        'Authorization': 'Bearer ' + store.state.token
+                    }
+                }
+            )
+            .then((res)=>{
+                this.pumps = res.data;
+                console.log(this.pumps);
```
Comment (Tj33rd) [inline Dashboard/resources/js/components/PumpsView/PumpsList.vue:30]:
```suggestion
```

File: Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+}
+</script>
+
+<template>
+<div class="p-4 m-4 bg-neutral-100 border divide-y divide-neutral-200 border-slate-200 rounded-sm max-h-screen">
+ <h1 class="font-bold text-3xl text-left bg-neutral-100 p-4">Gegevens van {{pumpdata.pump_name}}</h1>
+    <table class="table-auto w-full text-sm text-left text-gray-500">
+        <thead class="text-xs text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
+            <tr class="p-4">
+                <th scope="col" class="px-6 py-3">soort</th>
```
Comment (Tj33rd) [inline Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue:28]:
```suggestion
                <th scope="col" class="px-6 py-3">Soort</th>
```

File: Dashboard/resources/js/components/Mapbox.vue [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 <script>
 import leaflet from 'leaflet';
-import { onMounted } from '@vue/runtime-core';
+//import { onMounted } from '@vue/runtime-core';
```
Comment (streikbaut) [inline Dashboard/resources/js/components/Mapbox.vue:8]: Removed and then readded but commented out?

File: Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                <td class="px-6 py-4"> druk CV </td>
+                <td class="px-6 py-4">{{pumpdata.dataset.pressure_CV}}</td>
+            </tr>
+            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                <td class="px-6 py-4"> druk bron </td>
+                <td class="px-6 py-4">{{pumpdata.dataset.pressure_source}}</td>
+            </tr>
+            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                <td class="px-6 py-4"> druk verdamper </td>
```
Comment (Tj33rd) [inline Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue:42]:
```suggestion
                <td class="px-6 py-4">Druk verdamper</td>
```

File: Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+                <td class="px-6 py-4">{{pumpdata.dataset.pressure_evaporator}}</td>
+            </tr>
+            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                <td class="px-6 py-4"> druk condensor </td>
+                <td class="px-6 py-4">{{pumpdata.dataset.pressure_condensor}}</td>
+            </tr>
+            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                <td class="px-6 py-4">temp bron in</td>
```
Comment (Tj33rd) [inline Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue:50]:
```suggestion
                <td class="px-6 py-4">Temp bron in</td>
```

File: Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+                <td class="px-6 py-4">{{pumpdata.dataset.temp_source_into}}</td>
+            </tr>
+            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                <td class="px-6 py-4">temp bron uit</td>
```
Comment (Tj33rd) [inline Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue:54]:
```suggestion
                <td class="px-6 py-4">Temp bron uit</td>
```

File: Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+                <td class="px-6 py-4">{{pumpdata.dataset.temp_source_out}}</td>
+            </tr>
+            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                <td class="px-6 py-4">temp vorstbeveiliging brine</td>
```
Comment (Tj33rd) [inline Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue:58]:
```suggestion
                <td class="px-6 py-4">Temp vorstbeveiliging brine</td>
```

File: Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+                <td class="px-6 py-4">{{pumpdata.dataset.temp_frost_security_brine}}</td>
+            </tr>
+            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                <td class="px-6 py-4">temp stooklijn voetpunt </td>
```
Comment (Tj33rd) [inline Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue:62]:
```suggestion
                <td class="px-6 py-4">Temp stooklijn voetpunt</td>
```

File: Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+                <td class="px-6 py-4">{{pumpdata.dataset.temp_heating_curve_starting_point}}</td>
+            </tr>
+            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                <td class="px-6 py-4">temp stooklijn eindpunt </td>
```
Comment (Tj33rd) [inline Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue:66]:
```suggestion
                <td class="px-6 py-4">Temp stooklijn eindpunt</td>
```

File: Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+                <td class="px-6 py-4">{{pumpdata.dataset.temp_heating_curve_end_point}}</td>
+            </tr>
+            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                <td class="px-6 py-4">temp boiler </td>
```
Comment (Tj33rd) [inline Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue:70]:
```suggestion
                <td class="px-6 py-4">Temp boiler</td>
```

File: Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+                <td class="px-6 py-4">{{pumpdata.dataset.temp_boiler}}</td>
+            </tr>
+
```
Comment (Tj33rd) [inline Dashboard/resources/js/components/SinglePumpView/PumpData-chart-table.vue:73]:
```suggestion
```

File: Dashboard/resources/js/views/SinglePumpView.vue [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+export default {
+    components: {
+        Title,
+        Pumptable
+    },
+
+    methods: {
+        getPumpData(){
+            axios.get(
+                `http://localhost:8000/api/v1/pump/${this.$route.params.id}`,{
```
Comment (Tj33rd) [inline Dashboard/resources/js/views/SinglePumpView.vue:16]:
```suggestion
               process.env.MIX_API_URL + 'pump/${this.$route.params.id}`,{
```

File: Dashboard/resources/js/views/SinglePumpView.vue [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+                    headers: {
+                        'Authorization': 'Bearer ' + store.state.token
+                    }
+                }
+            )
+            .then((res)=>{
+                this.pump = res.data.data[0];
+                console.log(this.pump);
```
Comment (Tj33rd) [inline Dashboard/resources/js/views/SinglePumpView.vue:24]:
```suggestion
```

File: Dashboard/resources/js/views/SinglePumpView.vue [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+            });
+        }
+    },
+
+    data() {
+        return {
+            pump:{
+                dataset: {}
+            }
+
```
Comment (Tj33rd) [inline Dashboard/resources/js/views/SinglePumpView.vue:37]:
```suggestion
```

**Review by Tj33rd — CHANGES_REQUESTED** [review]

**Review by Tj33rd — APPROVED** [review]

**Pull request merged**